### PR TITLE
Update log code `schema-not-found`, `uid-conflict`

### PR DESCRIPTION
**Pull request by v-pegao**
| Log Code | Changes |
|----------|---------------|
| schema-not-found | message |
| uid-conflict | message |

### CODE REVIEW

File: src/docfx/Errors.cs [hunk 1]
```diff
@@ -404,7 +404,7 @@ public static Error UidConflict(string uid, IEnumerable<string> conflicts = null
             }
 
             var hint = conflicts.Count() > 5 ? "(Only 5 duplicates displayed)" : "";
-            return new Error(ErrorLevel.Error, "uid-conflict", $"Two or more documents have defined the same Uid '{uid}': {string.Join(',', conflicts.Take(5))}{hint}");
+            return new Error(ErrorLevel.Error, "uid-conflict", $"UID '{uid}' is defined in more than one file: {string.Join(',', conflicts.Take(5))}{hint}");
```
Comment (yufeih) [inline src/docfx/Errors.cs]: consider replace all the `string.Join` in `Errors.cs` with `Join` method to ensure all values a quoted.

File: src/docfx/Errors.cs [hunk 2]
```diff
@@ -404,7 +404,7 @@ public static Error UidConflict(string uid, IEnumerable<string> conflicts = null
             }
```
Comment (yufeih) [inline src/docfx/Errors.cs]: there is another case in line 403 where multiple UID is defined in the same file

File: src/docfx/Errors.cs (hunk 1 above)
Comment (v-pegao) [reply src/docfx/Errors.cs]: Updated. Only `circular-reference` is using `string.Join()` in `Errors.cs` now

File: src/docfx/Errors.cs (hunk 2 above)
Comment (v-pegao) [reply src/docfx/Errors.cs]: I didn't find the new message for the case in line 403 and the new message is not fit for the case in line 403 as it mentioned `more than one file`. Let me know if It needs update.

Comment (yufeih) [reply src/docfx/Errors.cs]: It needs update, but probably not in this PR, @mingwli this is another item to confirm

**Comment by OsmondJiang** [comment]
you need merge v3 firstly

**Pull request merged**